File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.js
function solution(num_list) {
  let result = 0;
  num_list.map(x => {
    let current = x;
    while (current > 1) {
      if (current % 2 === 1) {
        current = (current - 1) / 2;
      } else {
        current = current / 2;
      }
      result += 1;
    }
  })
  return result;
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        for(int num : num_list) {
            int answer2 = 0;
            while (num != 1) {
                if(num % 2 ==0) {
                    num /= 2;
                }else{
                    num = (num -1) /2;
                }
                answer2++;
            }
            answer += answer2;
        }
        return answer;
    }
}